algorithms/CSharp/src/Search/linear-search.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Algorithms.Search
+{
+    public class LinearSearch
+    {
+        public static void Main()
+        {
+            int[] numbers = { 1, 2, 3, 4, 5 };
+            int searchNumber = 3;
+            int indexOfSearchNumber = Search(numbers, searchNumber);
+
+            if (indexOfSearchNumber == -1)
+                Console.WriteLine("The number you are searching for does not exist in this list.");
+            else
+                Console.WriteLine("The number you are searching for is at index: " + indexOfSearchNumber);
```

Comment:
Please add the missing curly bracket
```suggestion
            if (indexOfSearchNumber == -1)
            {
                Console.WriteLine("The number you are searching for does not exist in this list.");
            }
            else
            {
                Console.WriteLine("The number you are searching for is at index: " + indexOfSearchNumber);
            }
```